pom.xml

```diff
@@ -57,7 +60,7 @@
 		<dependency>
 			<groupId>com.google.truth</groupId>
 			<artifactId>truth</artifactId>
-			<version>1.0.1</version>
+			<version>1.1.3</version>
```

Comment:
why was this changed?

Reply:
because it can be increased and it should be updated.